data.sql

```diff
@@ -1,14 +1,35 @@
 /* Populate database with sample data. */
 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (1, 'Agumon', '2020-02-03', 0, true, 10.23)
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Agumon', '2020-02-03', 0, true, 10.23)
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (2,  'Gabumon', '2018-11-15', 2, true, 8) 
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Gabumon', '2018-11-15', 2, true, 8) 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (3,  'Pikachu', '2021-01-07', 1, false, 15.0) 
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Pikachu', '2021-01-07', 1, false, 15.0) 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (4,  'Devimon', '2017-06-12', 5, true, 11)
\ No newline at end of file
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Devimon', '2017-06-12', 5, true, 11)
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Charmander', '2020-02-08', 0, False, 11);
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Plantmon', '2021-11-15', 2, True, 5.7);
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Squirtle', '1993-04-02', 3, False, 12.13);
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Angemon', '2005-06-12', 1, True, 45);
+
```

Comment:
- The weight of "Angemon" is `-45kg` not 45kg

![image](https://user-images.githubusercontent.com/75126481/220426457-13ba8323-8122-44d4-bc87-e7ea75be3d3d.png)